api/v1beta1/machineinventory_types.go

```diff
@@ -29,6 +29,7 @@ var (
 	PlanTypeBootstrap                                      = "bootstrap"
 	PlanTypeReset                                          = "reset"
 	MachineInventoryResettableAnnotation                   = "elemental.cattle.io/resettable"
```

Comment:
gofmt complaining: the whole var section should have values moved by one space more (aligned from 'corev1.SecretType'). Quickly fixable with:
`gofmt -e -d -s -w api/v1beta1/machineinventory_types.go`